### ISSUE TITLE
Publish/unpublish button problem in sorting mode 5

### ISSUE THREAD

**Issue by qzminski** [contao/core#8242]
In sorting mode 5 it seems that the toggle visibility button requires the image in record label to have a surrounding ```<a>``` tag or otherwise it will throw an error:

https://github.com/contao/core/blob/3.5.8/assets/contao/js/core-uncompressed.js#L530

I have a tree module in my extension that does have an icon next to each row but it is not wrapped by link (unlike to tl_page). Please allow to have this HTML markup:

```html
<div class="tl_left" style="padding-left:40px">
  <img src="assets/contao/images/iconPLAIN.gif" width="18" height="18" alt="" style="margin-left:20px"> Rock
</div>
```

and not force the link on the image:

```html
<div class="tl_left" style="padding-left:40px">
  <a href="#"><img src="assets/contao/images/iconPLAIN.gif" width="18" height="18" alt="" style="margin-left:20px"></a> Rock
</div>
```

See https://github.com/codefog/contao-news_categories/issues/81 for the reference.

@fritzmg

**Comment by leofeyer** [comment]
Would it solve your problem if I changed the line to the following?

```js
if (pa && pa.href.indexOf('do=feRedirect') == -1) {
```

**Comment by qzminski** [comment]
Yup, that would work just fine.

**Comment by leofeyer** [comment]
Fixed in 08e1e51707ef7607a91ff2423e2aaa3980f661d5 then.
